src/core_modules/capture-core/components/WidgetReferral/FormComponents/DateFieldForReferral.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+import React, { useState } from 'react';
+import { convertStringToDateFormat } from '../../../utils/converters/date';
+import {
+    DateField,
+    withDefaultFieldContainer,
+    withDisplayMessages,
+    withInternalChangeHandler,
+    withLabel,
+} from '../../FormFields/New';
+import labelTypeClasses from '../../WidgetEnrollmentEventNew/DataEntry/dataEntryFieldLabels.module.css';
```

Comment:
Can we duplicate what we need from here and put it inside the WidgetReferral folder hierarchy?